driver/ni/digital_read.cpp

```diff
@@ -22,129 +22,153 @@
 
 using json = nlohmann::json;
 
-void ni::DigitalReadSource::parseChannels(config::Parser &parser){
-    LOG(INFO) << "[NI Reader] Parsing Channels for task " << this->reader_config.task_name;
+void ni::DigitalReadSource::parseChannels(config::Parser &parser) {
+    LOG(INFO) << "[NI Reader] Parsing Channels for task " << this->reader_config.
+            task_name;
     // now parse the channels
     parser.iter("channels",
-                [&](config::Parser &channel_builder){
+                [&](config::Parser &channel_builder
+        ) {
                     ni::ChannelConfig config;
-
                     // digital channel names are formatted: <device_name>/port<port_number>/line<line_number>
-                    config.name = (this->reader_config.device_name + "/port" + std::to_string(channel_builder.required<std::uint64_t>("port")) + "/line" + std::to_string(channel_builder.required<std::uint64_t>("line")));
-
+                    config.name = (this->reader_config.device_name + "/port" +
+                                   std::to_string(
+                                       channel_builder.required<std::uint64_t>("port"))
+                                   + "/line" +
+                                   std::to_string(
+                                       channel_builder.required<std::uint64_t>(
+                                           "line")));
                     config.channel_key = channel_builder.required<uint32_t>("channel");
-
-                    // TODO: there could be more than 2 state logic
```

Comment:
Comments for top of file -> sort mports, remove extra attribution comments, and clean up whitespace